pkg/types/message_envelope.go

```diff
@@ -36,29 +37,29 @@ const (
 
 // MessageEnvelope is the data structure for messages. It wraps the generic message payload with attributes.
 type MessageEnvelope struct {
+	common.Versionable
+	// RequestID is an object id to identify the request.
```

Comment:
this comment doc should be in line 46 below

Reply:
fixed